Assets/HoloToolkit/Input/Scripts/Utilities/Managers/MixedRealityTeleport.cs

```diff
@@ -66,7 +66,7 @@ private void Start()
                 return;
             }
 
-            if (TeleportMarker != null)
+            if (teleportMarker != null)
```

Comment:
Please revert. This is checking that the prefab is set in the inspector, then it gets instantiated. This will never be called as written, since teleportMarker gets set as a result of this check and will thus alway be null here.

Reply:
It works.

Reply:
Not really sure why we're checking if it's null to begin with.  It'll throw errors later on if it's skipped.

Reply:
It was added in #1122. I see now that you made other changes that didn't show up in this PR, changing from two [tT]eleportMarker variables to one.

Reply:
Question: Is the teleport marker optional?  Do we want users to teleport without seeing one?

In either case the change is valid and works like you described.  I updated the private field to be serialized so you're setting the prefab on the private field instead of having a public and private field.

Reply:
For some reason GitHub isn't actually showing all the changes but it's there.
![image](https://user-images.githubusercontent.com/13334553/31525573-0f96b982-af76-11e7-9986-5cb6f77f44a5.png)

Reply:
I'd say no on the teleport marker being optional. Users should know that teleporting is a possibility before it happens.